.github/workflows/tests.yml

```diff
@@ -23,41 +23,42 @@ jobs:
           - macos-latest
           - ubuntu-latest
           - windows-latest
+        mode:
+          - debug
+          - release
     runs-on: ${{ matrix.os }}
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v3
         with:
           ref: ${{ github.event.pull_request.head.sha }}
-      - name: Tests debug build
+      - uses: dtolnay/rust-toolchain@stable
+        with:
+          targets: i686-pc-windows-msvc, i686-unknown-linux-gnu
+      - uses: Swatinem/rust-cache@v2
+
+      - name: Tests debug
+        if: matrix.mode == 'debug'
```

Comment:
That was my first guess but it turn out you can't `cargo test --debug`. And I doubt github action would welcome an empty string as a matrix variant.
So I guess this is the only way to do it

Reply:
Right, you have to put the entire argument in there, but that's certainly doable.
One option would be to have something like this
```
    - run: echo "TEST_MODE=" > $GITHUB_ENV
      if: ${{ matrix.mode == 'debug' }}
      shell: bash
    - run: echo "TEST_MODE=--release" > $GITHUB_ENV
      if: ${{ matrix.mode == 'release' }}
      shell: bash
```
and then
```
run: cargo test $TEST_MODE -p openmls --verbose
```
It would just be nice not to have all these jobs that aren't run.

Reply:
You're right, I've done something like without ghost steps.